examples/lock-app/nrfconnect/main/main.cpp

```diff
@@ -59,7 +59,6 @@ int main()
         goto exit;
     }
 
-#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
```

Comment:
Why is this un-necessary? I get that nrfConnect is mostly thread now, but why not leave this a config?

Reply:
I can't think of a reason why one would want to run the example without support for Thread now as it's the only way to operate in a CHIP network for nRF Connect platform. Also the code probably wouldn't compile without Thread anyway as it was only used at the very beginning of the development when the platform integration layer was incomplete. Examples for other platforms don't offer such an option either.